File: 2112-minimum-difference-between-highest-and-lowest-of-k-scores/2112-minimum-difference-between-highest-and-lowest-of-k-scores.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        nums.sort()
        mi = float('inf')
        for i in range(len(nums) - k + 1):
            d = nums[i + k - 1] - nums[i]
            if d < mi:
                mi = d
        return mi

File: 2112-minimum-difference-between-highest-and-lowest-of-k-scores/2112-minimum-difference-between-highest-and-lowest-of-k-scores.java
class Solution {
    public int minimumDifference(int[] nums, int k) {
        Arrays.sort(nums);
        int min = Integer.MAX_VALUE;
        for (int i = 0; i < nums.length - k + 1; i++) {
            int d = nums[i + k - 1] - nums[i];
            if (d < min) {
                min = d;
            }
        }
        return min;
    }
}